redis/tests/test_blocking_commands.rs

```diff
@@ -0,0 +1,32 @@
+#![cfg(feature = "cluster-async")]
+mod support;
+
+#[cfg(test)]
+mod test_blocking_commands {
+    use redis::{AsyncCommands, RedisResult, Value};
+
+    use crate::support::*;
+    #[tokio::test]
+    async fn test_blocking_command_when_cluster_drops() {
```

Comment:
why to invent a different logic and form - copy paste and modify:  https://github.com/amazon-contributing/redis-rs/blob/4571c91e2165b9ebf3bb9a3dff0991bc453d1075/redis/tests/test_cluster_async.rs#L3680

Reply:
you can add your test below that one